libraries/AP_DroneCAN/AP_DroneCAN.cpp

```diff
@@ -138,7 +138,14 @@ const AP_Param::GroupInfo AP_DroneCAN::var_info[] = {
     // @Range: 1024 16384
     // @User: Advanced
     AP_GROUPINFO("POOL", 8, AP_DroneCAN, _pool_size, DRONECAN_NODE_POOL_SIZE),
-    
+
+    // @Param: ESC_RV
+    // @DisplayName: ESC channels that support reverse
+    // @Description: Bitmask with one set for each channel that supports both forward and reverse
+    // @Bitmask: 0: ESC 1, 1: ESC 2, 2: ESC 3, 3: ESC 4, 4: ESC 5, 5: ESC 6, 6: ESC 7, 7: ESC 8, 8: ESC 9, 9: ESC 10, 10: ESC 11, 11: ESC 12, 12: ESC 13, 13: ESC 14, 14: ESC 15, 15: ESC 16, 16: ESC 17, 17: ESC 18, 18: ESC 19, 19: ESC 20, 20: ESC 21, 21: ESC 22, 22: ESC 23, 23: ESC 24, 24: ESC 25, 25: ESC 26, 26: ESC 27, 27: ESC 28, 28: ESC 29, 29: ESC 30, 30: ESC 31, 31: ESC 32
+    // @User: Advanced
+    AP_GROUPINFO("ESC_RV", 9, AP_DroneCAN, _esc_rev_bm, 0),
```

Comment:
I think `ESC_3D` would be a better fit for the pattern else where. That is a motor that we expect to command in both forwards and reverse. Typically reverse is to set the rotation direction and then use as normal.

Reply:
By pattern you're meaning I should rename the parameter to "CAN_*_ESC_3D" to match DShot and co, rather than implementing some higher-level 3D config?

Reply:
Right, just the param name.

Reply:
Changed.